<launch>
    <node pkg="object_finder" type="object_finder" name="object_finder_default" output="screen">
        <param name="use_probability" value="false"/>
        <rosparam file="$(find object_finder)/params/costmap_params.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_confidence" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_params.yaml" command="load" ns="costmap"/>
    </node>
    <!--node pkg="object_finder" type="object_finder" name="object_finder_laser_only" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_laser_only.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_laser_marking_only" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_laser_marking_only.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_laser_marking_only_inflation" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_laser_marking_only_inflation_10.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_headcam_only" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_headcam_only.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_inflation_10" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_inflation_10.yaml" command="load" ns="costmap"/>
    </node>
    <node pkg="object_finder" type="object_finder" name="object_finder_inflation_20" output="screen">
        <rosparam file="$(find object_finder)/params/costmap_inflation_20.yaml" command="load" ns="costmap"/>
    </node-->
</launch>